src/inputs/Table/components/TableToolbar/TableToolbar.spec.js

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { TableToolbar } from './TableToolbar';
+import { renderInMuiThemeProvider } from '../../../../../tests/utils';
+import { StackContainerTesting } from '../../../../../tests/MuiComponentsTesting';
+
+const tableToolbarInputContainer = new StackContainerTesting({ dataCy: 'table-toolbar' });
+const FullscreenButtonContainer = new StackContainerTesting({ dataCy: 'grid-fullscreen-button' });
+const ImportButtonContainer = new StackContainerTesting({ dataCy: 'import-file-button' });
+const ExportButtonContainer = new StackContainerTesting({ dataCy: 'export-button' });
+const AddRowButtonContainer = new StackContainerTesting({ dataCy: 'add-row-button' });
+const DeleteRowButtonContainer = new StackContainerTesting({ dataCy: 'delete-rows-button' });
+
+const defaultProps = {
+  isFullscreen: false,
+  toggleFullscreen: () => {},
```

Comment:
it's better to use `jest.fn() `to mock function in props (e.g. _src/inputs/SelectWithAction/SelectWithAction.spec.js_ line 10 or 11)
same remark for `onImport` `onExport`, `onAddRow` and `onDeleteRow` functions

Reply:
DONE